frontend/src/components/CountdownDigital.tsx

```diff
@@ -1,72 +1,131 @@
-import { useQueryClient } from "@tanstack/react-query";
 import type React from "react";
-import { useEffect } from "react";
-import { useEpisodeId } from "../contexts/EpisodeContext";
-import { buildKey } from "../api/helpers";
-import { nextTournamentFactory } from "../api/episode/episodeFactories";
-import {
-  HOURS_DAY,
-  MILLIS_SECOND,
-  MINUTES_HOUR,
-  SECONDS_MINUTE,
-} from "utils/utilTypes";
+import { useEffect, useMemo, useState } from "react";
 
-interface CountdownDigitalProps {
-  date: Date;
-  title?: string;
+const thousand = 1000;
+const sixty = 60;
+const hoursInDay = 24;
```

Comment:
I think the original constants for these from utils are sufficient and perhaps more clearly specify their purpose (i.e. MILLIS_SECONDS == milliseconds per second)